lib/APPTurtle/src/App.cpp

```diff
@@ -55,6 +57,11 @@
  * Types and classes
  *****************************************************************************/
 
+/**
+ * Subscriber for Geometry Twist ROS messages.
+ */
+typedef Subscriber<geometry_msgs__msg__Twist> TwistSubscriber;
```

Comment:
Is this really the name to use ? Maybe use local typedef to turn int into something pretty.

Reply:
Name is not pretty, as it is only used to create the subscription. Still made it local